package app.telegram.bot.exceptions.user;

public class TaskAlreadyExistsException extends RuntimeException {
}
